app/scripts/modules/core/src/application/modal/PermissionsConfigurer.spec.tsx

```diff
@@ -0,0 +1,56 @@
+import {mock} from 'angular';
+import * as React from 'react';
+import {mount} from 'enzyme';
+import {ReactInjector, REACT_MODULE} from 'core/reactShims';
+
+import {
+  IPermissions, IPermissionsConfigurerProps,
+  PermissionsConfigurer
+} from 'core/application/modal/PermissionsConfigurer';
+import {AUTHENTICATION_SERVICE} from 'core/authentication';
```

Comment:
space around `{` and `}`

(I know it's hard to keep up with the ever-changing code format standards)